docs/eog/case-studies-cw20-event-roles.rst

```diff
@@ -0,0 +1,32 @@
+
+- Event Chair/Facilitator (filled in this case by the Event Lead)
+- A Code of Conduct committee
+- A (Slack) help desk for technical issues
+- A general Slack monitor for questions and engagement
+- A Twitter monitor for tweeting about the event
+- Session chairs
+- Zoom participant managers for un/muting, creating/assigning breakout rooms, keeping an eye on hand raising, etc.
+- Four sets of Hosts and Co-hosts for the parallel Mini-workshop and demo sessions
+- Hackday judges
+
+What wasn’t initially clear, was the effect that the government response to COVID-19 would have on how staff could participate in the event. 
+Parents now had to factor in homeschooling kids and other caring responsibilities during the day. 
+Other team members had limited or unstable internet, and therefore wouldn’t be able to have prominent Zoom responsibilities. 
+Some members became ill. 
+We therefore ensured that we had two people assigned for each role/duty, as contingency for if someone had to drop out of the event for any reason. 
+This required us to reach out to our network, as there weren’t enough SSI staff available to cover every role and backup. 
+Luckily, the research software community has no shortage of virtual event experts, so we reached out to our SSI Fellows in the `Open Life Science <https://openlifesci.org/>`_ community to help us manage the parallel mini-workshop and demo sessions.
+
+We documented everything to make sure that everyone knew their roles and responsibilities: we created a `duties roster <https://doi.org/10.6084/m9.figshare.12498122>`_, wrote `instructions for Zoom Hosts and Co-hosts <https://doi.org/10.6084/m9.figshare.12498191>`_, created a `spreadsheet of the various roles and Zoom room links for the parallel sessions <https://doi.org/10.6084/m9.figshare.12498164>`_, and provided an `index of links <https://doi.org/10.6084/m9.figshare.12459656>`_ to point to all these documents and to help people find anything else related to CW20 that they would need (links to each day’s notes, Zoom rooms, communication channels, session information, etc.).
+ 
+We organised test calls for CW20 helpers and Zoom Hosts to make sure all the necessary features were enabled for each account (breakout rooms, co-hosts, non-verbal feedback, etc.) and to let the Hosts gain experience using these features. 
+We also scheduled Zoom “Office Hours” for workshop facilitators to join and test the functionality of Zoom in preparation for their sessions and to ask any questions. 
+Finally, we had a Code of Conduct Committee call to walk through the incident response procedure, discuss potential violations that may occur during an online event and any additional risk mitigation we could put in place to prevent them. 
+
+You can find the full collection of `CW20 infrastructure and documentation resources on Figshare <https://doi.org/10.6084/m9.figshare.c.5026400>`_.
```

Comment:
Again - it seems a bit strange to put this here - I can understand if someone came in via a google search let's say - perhaps add a sub-heading - e.g. resources. I can see why you put this here - but it's a bit jarring as the last sentence - but then again perhaps we can have a chat about what we want on each page and it is a bit weird with guides done this way - each page has to almost standalone.